flex/engines/hqps_db/core/operator/sink.h

```diff
@@ -175,13 +176,80 @@ void template_set_tuple_value(results::Collection* collection,
   }
 }
 
+/////Sink Any
+
+void set_any_to_common_value(const Any& any, common::Value* value) {
+  switch (any.type) {
+  case gs::PropertyType::kInt32:
+    value->set_i64(any.value.i);
+    break;
+  case gs::PropertyType::kInt64:
+    value->set_i64(any.value.l);
+    break;
+  case gs::PropertyType::kDate:
+    value->set_i64(any.value.d.milli_second);
+    break;
+  case gs::PropertyType::kString:
+    value->mutable_str()->assign(any.value.s.data(), any.value.s.size());
+    break;
+  case gs::PropertyType::kDouble:
+    value->set_f64(any.value.db);
+    break;
+  default:
+    LOG(FATAL) << "unknown type";
```

Comment:
Better with `unsupported` and log any.type as well. `Log(Warn)` instead of FATAL?

Reply:
Fixed.